static/app/views/dashboardsV2/datasetConfig/errorsAndTransactions.tsx

```diff
@@ -57,6 +64,21 @@ export const ErrorsAndTransactionsConfig: DatasetConfig<
   transformTable: transformEventsResponseToTable,
 };
 
+function getEventsTableFieldOptions(
+  contextualProps?: ContextualProps,
+  tags?: TagCollection
+) {
+  const organization = contextualProps?.organization!;
+  const measurements = {...WEB_MEASUREMENTS, ...MOBILE_MEASUREMENTS};
```

Comment:
I think it might be nicer is `measurements.tsx` exposed a function for this instead of manually constructing it since if it ever gets updated we might forget about this spot